Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM instructure/node:12
+FROM instructure/node:14
```

Comment:
Node 14 was EOL 2023-04-30. Why not upgrading to Node 20 at least? There is an instructure/node:20 already available.